Bio/Blast/NCBIWWW.py

```diff
@@ -275,6 +276,13 @@ def qblast(
         if delay < 60 and url_base == NCBI_BLAST_URL:
             # Wasn't a quick return, must wait at least a minute
             delay = 60
+        elapsed = time.time() - start_time
+        # Throw a warning if search takes longer than ten minutes
+        if elapsed >= 600:
+            warnings.warn(
+                "Search is taking longer than 10 minutes, consider re-issuing it",
+                BiopythonWarning,
+            )
```

Comment:
This warning will be issued only once ...

Reply:
Dear @mdehoon,

I guess that, from what I can understand from the code (and from what can be easily seen if you set something like "if elapsed >= 0"), the issue should be thrown at the end of every pooling cycle (so approx. every minute), after the program has hit the 10 minutes threshold. I don't know if I am getting something wrong, but when I tried it, it worked and it was issued more than once: let me know if this wasn't the case for you! 

All the best,
Astra

Reply:
If you call `qblast` twice within the same Python session, do you get the warning twice?
(assuming that both calls to qblast take more than 10 minutes).